### DISCUSSION TITLE
How to associate more than one app with an entry?

### DISCUSSION THREAD

**Discussion by kevinlucasilva** [Kunzisoft/KeePassDX#1828] [Q&A]
I have already defined the ``AndroidApp`` field for an entry, but it is possible to enter it in another app. How do I associate another one?

In KeePassXC it is possible to add more URLs using the ``K2PA_URL`` field, then ``K2PA_URL_1`` and so on., so I tried adding the ``AndroidApp_1`` field, but it didn't work.

**Comment by J-Jamet** [comment]
K2PA notation is not recommended, because specific to KeePass2android.
https://github.com/Kunzisoft/KeePassDX/wiki/AutoFill#application
AndroidApp_1 should work.
